pages/common/black.md

```diff
@@ -0,0 +1,24 @@
+# black
+
+> A speedy Python auto code formatter.
+> More information: <https://github.com/psf/black>.
+
+- Auto-format a file or entire directory:
+
+`black {source_file_or_directory}`
```

Comment:
Looks like this should be using double curly braces to match the style guide, could you also update the references to use `path/to/file_or_directory` throughout the page? 🙂

```suggestion
`black {{path/to/file_or_directory}}`
```